packages/gatsby-plugin-manifest/src/gatsby-node.js

```diff
@@ -1,6 +1,6 @@
 import fs from "fs"
 import path from "path"
-import sharp from "sharp"
+import sharp from "./safe-sharp"
```

Comment:
I will apply this to any `sharp` import in monorepo. Hopefully this will not stay long, so I don't want to create separate package for it and will just copy&paste that file around. The ultimate plan (right now) is to make `sharp` peerDependency of those packages, but this is pretty disruptive change so I want to schedule this for gatsby@3. In case those files would stay around - we can refactor them into separate package later on aswell.